lib/coney/rabbit_connection.ex

```diff
@@ -32,30 +32,41 @@ defmodule Coney.RabbitConnection do
 
     Basic.qos(chan, prefetch_count: connection.prefetch_count)
 
-    exchange_name = declare_exchange(chan, connection.exchange)
     queue = declare_queue(chan, connection.queue)
 
-    Queue.bind(chan, queue, exchange_name, Map.get(connection, :binding, []))
+    exchange_name = declare_exchange(chan, Map.get(connection, :exchange, nil))
+
+    bind_queue(chan, exchange_name, queue, Map.get(connection, :binding, []))
 
     {:ok, _consumer_tag} = Basic.consume(chan, queue, consumer_pid)
   end
 
+  defp declare_queue(chan, {name}) do
+    declare_queue(chan, {name, []})
+  end
+
+  defp declare_queue(chan, {name, params}) do
+    Queue.declare(chan, name, params)
+    name
+  end
+
   defp declare_exchange(chan, {type, name}) do
     declare_exchange(chan, {type, name, []})
   end
 
+  defp declare_exchange(_, {:direct, "", _}), do: :default_exchange
+  defp declare_exchange(_, nil), do: :default_exchange
```

Comment:
Let's leave `exchange` key required, otherwise it wont be obvious which exchange is being used by consumer. And it would be nice to update docs about `:default` option 🙂